pytest.ini

```diff
@@ -1,5 +1,5 @@
 [pytest]
-addopts = --cov=graspy
+#addopts = --cov=graspy
```

Comment:
please explain

Reply:
did not mean to push this, had to comment this out for pytest to run?

Reply:
You probably just need to install pytest-cov https://pypi.org/project/pytest-cov/